lib/contributors/github.js

```diff
@@ -15,7 +15,7 @@ module.exports = function getUserInfo(username, cb) {
     var body = JSON.parse(res.body);
     var user = {
       login: body.login,
-      name: body.name,
+      name: body.name === null ? username : body.name,
```

Comment:
Do you know if users can make their name be an empty string? If so, maybe `bode.name || username` would be a better fit?